src/main/webapp/script.js

```diff
@@ -73,7 +74,7 @@ function fetchList(queryString) {
  * @param {string} website The url of the business' website
  * @param {int} totalCardCount The number of businesses in the list, used to set a specific id to each card
  */
-function createResultCard(name, address, photos, rating, totalCardCount) {
+function createResultCard(name, address, photos, rating, website, totalCardCount) {
```

Comment:
nit: Please prefer something like websiteUrl to be more specific.

Reply:
fixed!